### ISSUE TITLE
add accessibility ids

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1449]
never use position based element queries, instead always use accessibility identifiers. Add them to codebase when missing.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/issues/1056#issuecomment-974681036_

**Comment by tomholub** [comment]
@fcvakintos please make a list of all test ui query situations when we are looking for "third element from the top" and similar (except when we deliberately do it with aid, like recipient bubbles - that is good). 

I've seen some before, forgot where. All of those should be replaced with specific aid in the app.

**Comment by fcvakintos** [comment]
@tomholub hey, do you mean something like these?
`NAME_AND_EMAIL: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[1]',
  DATE_CREATED: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]',
  FINGERPRINT: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[3]'`

**Comment by tomholub** [comment]
Yes, exactly. The [1] and [2] and [3] will give us trouble, the moment we add some element it will break these tests. Whereas with aid they will remain working even after we rearrange the UI.

**Comment by fcvakintos** [comment]
1. NAME_AND_EMAIL: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[1]',
2. DATE_CREATED: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]',
3.FINGERPRINT: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[3]',
4.  ENTER_PASS_PHRASE_FIELD: '-ios class chain:**/XCUIElementTypeSecureTextField', 

<img width="429" alt="Screenshot 2022-03-29 at 16 29 16" src="https://user-images.githubusercontent.com/91205853/160622580-43c8f21b-c933-496b-a304-74bcb664af57.png">
<img width="1277" alt="Screenshot 2022-03-29 at 16 37 48" src="https://user-images.githubusercontent.com/91205853/160624260-bd6ecea1-1d39-4cc1-b634-4fe0c2c96599.png">

**Comment by fcvakintos** [comment]
@tomholub check pls

**Comment by tomholub** [comment]
let's add accessibility ids to replace these, called `aid-name-and-email`, `aid-date-created` and so on.

    NAME_AND_EMAIL: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[1]',
    DATE_CREATED: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]',
    FINGERPRINT: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[3]',
    ENTER_PASS_PHRASE_FIELD: '-ios class chain:**/XCUIElementTypeSecureTextField',

**Comment by ioanmo226** [comment]
Please don't touch UI tests related to KeyScreen. It's already being updated in #841 .

> 
> let's add accessibility ids to replace these, called aid-name-and-email, aid-date-created and so on.
> 
> NAME_AND_EMAIL: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[1]',
> DATE_CREATED: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]',
> FINGERPRINT: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[3]',
> ENTER_PASS_PHRASE_FIELD: '-ios class chain:**/XCUIElementTypeSecureTextField',
>
